.azure-ci.yml

```diff
@@ -25,6 +25,9 @@ resources:
   - container: alpine_36_container
     image: microsoft/dotnet-buildtools-prereqs:alpine-3.6-WithNode-f4d3fe3-20181213005010
 
+  - container: alpine_37_arm64_container
```

Comment:
Starting to look at this from the Core-Setup side, and I realized this is called 3.7 instead of 3.6. Why is that? (I don't generally understand what makes the cross-build images a particular distro version.)

Reply:
That is exactly why I called it 37, because I also needed to figure out the version, so went and look how we build this cross images. I saw that we call a build-rootfs.sh script in core-setup. So for this image, we pass in `alpine` as the Linux code name. Which will pick 3.7 as the version.

https://github.com/dotnet/core-setup/blob/e9692c95240cf23da279fb3f69942591674e6d44/cross/build-rootfs.sh#L173

@jashook correct me if I'm wrong

Reply:
/cc @janvorli

Reply:
Sounds correct to me, although it is technically an ubuntu image. Just the rootfs it has is based on alpine 3.7

Reply:
Yeah, it is an ubuntu image, but since we use it to build for alpine and yesterday it came to my mind, what version of alpine did we use for the rootfs, then I stick it into the name so that if someone else asks the question while reading the files, it is self documented.